rpcs3/main.cpp

```diff
@@ -255,7 +255,7 @@ struct fatal_error_listener final : logs::listener
 
 	void log(u64 /*stamp*/, const logs::message& msg, const std::string& prefix, const std::string& text) override
 	{
-		if (msg == logs::level::fatal)
+		if (msg <= logs::level::fatal)
 		{
```

Comment:
Why this change?

Reply:
It's to debug stuff with logging in real time, not relying on the execution of main thread using the always level.